examples/usecases/accelerate-training-of-large-embedding-tables-by-LazyAdam.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Should the title be "using multiple optimizer to accelerate training by using lazyadam for large embeddings" or something like that? It misses the information that we use different optimizer?</p><p> </p><p>Should we name the results of our experiments -&gt; it seems you got almost a 2x speed up (30s vs 56s). I think we should highlight this, that is amazing.</p><p> </p><p>The headings are not consistent -&gt; sometimes they are larger, sometimes they are smaller -&gt; what are the main headings? what are the sub headings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
thanks. will fix these.

Reply:
@bschifferer  I get ~ 4X speed up: 18 ms vs 71 vs.